google/cloud/spanner/client.h

```diff
@@ -0,0 +1,193 @@
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_CLIENT_H_
```

Comment:
For external projects we do need the Copyright boilerplate, please add here and in any new files.